alpine.dockerfile

```diff
@@ -34,11 +34,9 @@ LABEL maintainer="Microsoft" \
 
 ARG JP_VERSION="0.2.1"
 
-WORKDIR azure-cli
-COPY . /azure-cli
```

Comment:
It is weird to use relative path `azure-cli` in `WORKDIR` and absolute path `/azure-cli` in `COPY`. This relies on the container's pwd being `/`.

Reply:
The `alpine` and `python-alpine` does not set `workdir`, so its value is `/` by default.